Formula/gwsocket.rb

```diff
@@ -0,0 +1,22 @@
+class Gwsocket < Formula
+  desc "Simple, standalone, RFC6455 compliant WebSocket server"
+  homepage "http://gwsocket.io/"
+  url "https://github.com/allinurl/gwsocket/archive/v0.2.tar.gz"
+  sha256 "cafb0cbde1d8b366c74c1602a8d2e1bde054834aa5c618cfad5603a871ae4082"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+
+  def install
+    system "autoreconf", "-fvi"
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/gwsocket", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!